### PR TITLE
Add facilites + working tests

### PR DESCRIPTION
**Pull request by Ephraim-Kamau**
**Description**
I have added the facility functionality and all its relevant files to the other functionalitites.

**How can This be tested**
- Clone the repo
- Setup project according to `Readme.md`
- checkout to branch ``
- 

**Type of change**
- [ ] Bugfix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have linted my code before submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

**AZURE**
[ID](link)

**Relevant screenshots**

### CODE REVIEW

File: helpers/facilities_helper.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+facility_schema = {
+    'id': fields.Integer(description='The facility unique identifier'),
+    'name': fields.String(description='Facility name'),
+    'location': fields.String(description='Facility location'),
+    'certifications': fields.String(description='Facility certifications'),
+    'phone_no': fields.String(description='Facility phone number'),
+    'email': fields.String(description='Facility email address'),
+    'status': fields.String(attribute=lambda x: x.status.value,
+                            description='Facility status'),
+}
```
Comment (Ephraim-Kamau) [inline helpers/facilities_helper.py]: this schema is replicated in `facility_schema.py`

File: migrations/versions/d133607c824c_create_roles_and_users_tables.py [hunk 2]
```diff
@@ -23,33 +21,41 @@ def upgrade():
...
     op.create_table('roles',
-    sa.Column('created_on', sa.DateTime(), nullable=True),
-    sa.Column('updated_on', sa.DateTime(), nullable=True),
-    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
-    sa.Column('role', sa.String(length=250), nullable=False),
-    sa.Column('code', sa.Integer(), nullable=False),
-    sa.PrimaryKeyConstraint('id'),
-    sa.UniqueConstraint('code'),
-    sa.UniqueConstraint('role')
-    )
```
Comment (Ephraim-Kamau) [inline migrations/versions/d133607c824c_create_roles_and_users_tables.py]: This change is not relevant to this task

File: api/models/facilities_model.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from passlib.context import CryptContext
```
Comment (mnswaleh) [inline api/models/facilities_model.py]: you don't need this import in your model

File: api/__init__.py [hunk 4]
```diff
@@ -2,3 +2,6 @@
 
 user_ns = Namespace("users", description="users operations")
 profiles_ns = Namespace("profiles", description="profile operations")
+patient_prof_ns = Namespace("patients", description="patient profiles")
+role_ns = Namespace("roles", description="user roles")
```
Comment (mnswaleh) [inline api/__init__.py]: These lines are not used anywhere in the project

File: api/models/facilities_model.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+class Facility(Base):
+    __tablename__ = 'facilities'
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(250), nullable=False)
+    location = db.Column(db.String(250), nullable=False)
+    certifications = db.Column(db.String(250), nullable=False)
+    phoneNo = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(250), nullable=False)
+    status = db.Column(db.Enum(StatusType),
+                       default=StatusType.active, nullable=False)
```
Comment (mnswaleh) [inline api/models/facilities_model.py]: create a new facility status type which will `pending`, `expired` and `blocked` depending on facility status

File: fixtures/__init__.py [hunk 6]
```diff
@@ -1,3 +1,4 @@
 from flask_restplus import Namespace
 
 user_ns = Namespace("users", description="users operations")
+facility_ns = Namespace("facilities", description="facilities")
```
Comment (mnswaleh) [inline fixtures/__init__.py]: remove these lines, they are not relevant in the project

**Pull request merged**